.github/workflows/verify.yaml

```
@@ -128,3 +139,22 @@ jobs:
 
       - name: Run Postgresql E2E tests
         run: ./gradlew test -DincludeTags="PostgresqlIntegrationTest" --refresh-dependencies
+
+  generate-allure-report:
+    runs-on: ubuntu-latest
+    needs: end-to-end-tests
+    if: ${{ github.ref_name == 'main' || startsWith(github.ref_name, 'bugfix/') || startsWith(github.event.pull_request.head.ref, 'release/') }}
```

currently releases are merged back into main so there's no need to check for specific pull requests

```suggestion
    if: ${{ github.ref_name == 'main' || startsWith(github.ref_name, 'bugfix/') }}
```

---

Agree with you on this.